client/core/core.go

```diff
@@ -900,10 +900,12 @@ func (c *Core) connectedDEX(addr string) (*dexConnection, error) {
 	return dc, nil
 }
 
-// setEpoch sets the epoch. If the passed epoch is greater than the highest
-// previously passed epoch, an epoch notification is sent to all subscribers and
-// true is returned.
-func (dc *dexConnection) setEpoch(mktID string, epochIdx uint64) bool {
+// tryUpdateEpoch updates the epoch for market with mktID to epochIdx,
+// sends an epoch notification to all subscribers
+// and returns true only
+// if the passed epoch is greater than the highest previously observed epoch.
+// Otherwise tryUpdateEpoch returns false.
+func (dc *dexConnection) tryUpdateEpoch(mktID string, epochIdx uint64) bool {
```

Comment:
This name change adds nothing and is completely unrelated to the task at hand. In general, do not change anything except what's necessary. We will sometimes accept an un-related or stylistic change as part of PR, but not until you've established that you can keep your work properly scoped.

Reply:
Right, should I remove this change ?

Reply:
Yeah, pls leave `setEpoch` and it's docs as-is. Thanks.

Reply:
Either way is fine with me... but another pr, even for just this one change is probably better.

Reply:
Will move this to a separate PR.

Reply:
No need. It doesn't improve clarity or readability, and even if it did, it's not something to spend time on right now.